channels-and-providers/push/pushpad.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: 'Pushpad'
+description: 'Learn how to use the Pushpad provider to send web push notifications using Novu'
+---
+
+[Pushpad](https://pushpad.xyz) is a web push service that supports sending notifications to all major browsers (Chrome, Firefox, Edge, Safari, etc.) via FCM, Mozilla autopush, Windows Push Notification Services and Apple Push Notification service, with just one simple API.
+
+To configure the Pushpad integration, you will need an active account and you need to have a `Pushpad Auth Token` (from the account settings) and the `Pushpad Project ID` (from the project settings).
+
+# Setting the Device Token
+
+Once Pushpad has been configured with your credentials and the Pushpad SDK has been [set up and installed](https://pushpad.xyz/docs/pushpad_pro_getting_started) on your website, you can [assign a user ID (uid)](https://pushpad.xyz/docs/identifying_users) to the push subscriptions.
+
+This identifier (`uid` or `user ID`) allows targeting a specific browser when sending push notifications.
+
+In order to target the Pushpad user from Novu, you must register the Pushpad `uid` as the `deviceToken` for your Novu subscriber. For example, if you invoked the [Pushpad JavaScript SDK](https://pushpad.xyz/docs/javascript_sdk_reference) with `pushpad('uid', 'user123')`, then `user123` is the user ID for that browser.
+
+Once you have the `uid` value, the `deviceToken` for your Novu subscriber can be set using this code:
+
+```jsx
+import { Novu, PushProviderIdEnum } from '@novu/node';
+
+const novu = new Novu("<NOVU_API_KEY>");
+
+await novu.subscribers.setCredentials('subscriberId', PushProviderIdEnum.Pushpad, {
+  // the user ID (uid) that you used for Pushpad
+  deviceTokens: ['user123'],
+});
+
+```
+
+# SDK Trigger Example
+
+```jsx
+import { Novu } from '@novu/node';
+
+const novu = new Novu("<NOVU_API_KEY>");
+
+novu.trigger('event-name', {
+  to: {
+    subscriberId: '...',
+  },
+  payload: {}
+});
+
+```
+
+Device/notification identifiers can be set by using [setCredentials](https://docs.novu.co/channels/email/resend/#set-device-token) or by using the `deviceIdentifiers` field in overrides.
```

Comment:
This is not a valid URL. Please can you change it?

Reply:
@unicodeveloper I got that link from the docs of other push providers, but I agree that it seems broken at the moment. I don't like the idea to link to another provider (push-webhook), that is confusing. I see that the example for `setCredentials ` is just in the paragraph above, so I removed the link and improved the sentence.